### DISCUSSION TITLE
Port Closed issue

### DISCUSSION THREAD

**Discussion by Bansantion** [Novik/ruTorrent#2365] [Q&A]
I have ruTorrent Setup in Docker on synology Nas using AIRVPN and for some reason at the bottom the little red circle says my port is closed attached is screenshot of my port settings can someone please help (my knowledge on this subject is very limited so please speak in layman terms if possible) Starting from top to bottom pictures is the red circle then rutorrent settings then the docker port settings followed by air vpn ports and what happens when i click test connection then my routers port forwarding settings

![9cbb58551ecb3edeb95af9827d53467e422562c0](https://user-images.githubusercontent.com/71607370/172070704-8f326f72-9d9c-42d1-8593-d5b49f476bd2.png)

![916189686816c35f57a0b79ecb7d29ab9ae3dbcb](https://user-images.githubusercontent.com/71607370/172070679-e5928850-9247-42cc-9f5b-7b0d632487f2.png)

![9da41d1fa0ea81841600342af988147821fb2723](https://user-images.githubusercontent.com/71607370/172070730-558b3c22-2455-44d6-98a9-a95b74d9538a.png)

![491c74ad596bc4690c71e22c7a96a643a7b700cf](https://user-images.githubusercontent.com/71607370/172070745-89c38e17-0e03-4d7e-8c46-656b611856e6.png)

![0db35defc14e5e88437969eeda97e41b666f2dce](https://user-images.githubusercontent.com/71607370/172070803-1e2fc4ef-0830-4036-9b28-cbd404f25d9d.png)
![ef825f2ca26645ac1b5600085f872065f4faf695](https://user-images.githubusercontent.com/71607370/172070822-5ea2c265-5f8b-44a0-a99a-52e81a4a0072.png)

![9a20c97768316ae83b571957300ee6ac771b12de](https://user-images.githubusercontent.com/71607370/172070768-1901cc45-339f-4959-9b26-e90419624106.png)

**Comment by stickz** [comment]
You have rTorrent setup to listen on a closed port range 6881-6999.

Try changing the port used for incoming connections to 58439-58439.
